includes/discovery/sensors/voltage/siklu.inc.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-$oid  = 'rbSysVoltage.0';
```

Comment:
The initial issue for this support indicates that OID is correct:

https://github.com/librenms/librenms/issues/798

Has the vendor broken this in a later version? This will break support for existing devices (or at least clean the data out for it to start again).

Reply:
Can you help me understand why the discovery process looks fine, but the poller fails?

Discovery:

```
Voltage: SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OsqnU' '-m' 'RADIO-BRIDGE-MIB' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/siklu' 'udp:HOSTNAME:161' 'rbSysVoltage.0']

SQL[SELECT * FROM sensors AS S, devices AS D WHERE S.sensor_class=? AND S.device_id = D.device_id AND D.device_id = ? AND S.poller_type = ? ["temperature",3,"snmp"] 0.64ms] 
  
.1.3.6.1.4.1.31926.1.2.0 -> siklu  

Siklu Voltage 
.*.4.1.319* 52  
```

Poller:

```
SNMP['/usr/bin/snmpget' '-v2c' '-c' 'COMMUNITY' '-OUQnte' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/siklu' 'udp:HOSTNAME:161' 'rbSysVoltage.0']

  

rbSysVoltage.0: Unknown Object Identifier (Sub-id not found: (top) -> rbSysVoltage)  

Checking (snmp) voltage System voltage... 
```

Reply:
If you run that same snmpget in CLI, does it return a result ? 

My guess : OID has to be numerical for the snmpget, so most probably there is something wrong when sensor is created.

Reply:
@PipoCanaja No - the snmpget fails when running from a shell as well.  It works with the OID, not with `rbSysVoltage.0`.

Reply:
@PipoCanaja @laf What can we do to get this one across the finish line?

Reply:
Let see how tests are going with these few changes. Should get to something mergeable soon.

Reply:
By the way, you probably have same issue with temperature and other sensors using the same structure of discovery code. If numerical OID is not passed to discover_sensor, there is no way it can make it during polling code, I would say.

Reply:
Temperature actually works just fine.  I know, weird, right?

Reply:
Temperature sensor discovery was actually changed between the original PR and today. That's why it works.

Reply:
Just checked it, 'Temperature' sensor discovery was actually changed between the original PR and today. That's why it works.